tests/test_tavern.py

```diff
@@ -4668,6 +4668,9 @@ def test_archdruid_hammul(self):
         self.upgrade_to_tier(tavern, 6)
         tavern.randomizer = RepeatedCardForcer([DeckSwabbie, ArchdruidHamuul])
         tavern.buying_step()
```

Comment:
I think we can just make the list of cards to force longer, so we will specify the full store, and the cards that Archduid draws as well.